runtime/karura/src/lib.rs

```diff
@@ -1250,6 +1252,13 @@ impl cumulus_pallet_xcm_handler::Config for Runtime {
 	type AccountIdConverter = LocationConverter;
 }
 
+pub struct HandleXcm;
+impl XcmHandlerT<AccountId> for HandleXcm {
```

Comment:
I think it is safe to move this into common?

Reply:
The impl needs `XcmHandler` from `construct_runtime` of each runtime. I think they are not accessible in common runtime.